server/services/offer.service.js

```diff
@@ -560,7 +560,7 @@ export const acceptOffer = async (offerToAccept) => {
       description: descriptionUser,
     });
 
-    const acceptedoffer = await getOffer(offerToAccept.offerId, offerToAccept.user);
+    const acceptedoffer = await getOffer(offerToAccept.offerId, { role: USER_ROLE.ADMIN });
```

Comment:
I think it will be better to add a third parameter to include sensitive information for user. With that, we would only need to confirm if the role is an admin or the sensitive parameter is set to true to return sensitive information. If a new developer is reading this, it would be so confusing why we are adding the `adminRole` to a normal user.

That way we will be able to return sensitive information when needed by the user or vendor without changing the user role. Something like this:

```suggestion
    const acceptedoffer = await getOffer(offerToAccept.offerId, offerToAccept.user, true);
```